spec/system/dashboards_spec.rb

```diff
@@ -136,27 +136,8 @@
                       20) + create_list(:prior_authority_application, 1, state: 'sent_back', updated_at: Date.yesterday - 1)
         end
         let(:endpoint) { 'https://appstore.example.com/v1/submissions/searches' }
-        let(:search_payload) do
-          {
-            application_type: 'crm4',
-            explicit_application_type: true,
-            page: 1,
-            per_page: 20,
-            sort_by: 'last_updated',
-            sort_direction: 'descending',
-          }
-        end
-        let(:sort_payload) do
-          {
-            application_type: 'crm4',
-            explicit_application_type: true,
-            page: 1,
-            per_page: 20,
-            sort_by: 'status_with_assignment',
-            sort_direction: 'ascending',
-          }
-        end
-
+        let(:search_payload) { {} }
```

Comment:
Do these lines do anything?

Reply:
no actually - thought I needed them as a stand in for populating the stubs outside of context but that isn't the case - removed them